File: 1580-shuffle-the-array/shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int x = 0;
        int y = n;
        int arr[] = new int[2*n];
        for (int i = 0; i < arr.length; i+=2){
            arr[i] = nums[x++];
            arr[i+1] = nums[y++];
        }return arr;
        
    }
}

File: 1580-shuffle-the-array/shuffle-the-array.py
class Solution(object):
    def shuffle(self, nums, n):
        x = 0
        y = n

        l = []
        for i in range(n):
            l.append(nums[x])
            l.append(nums[y])
            x += 1
            y += 1
        

        return l 
        